auction/tasks.py

```diff
@@ -25,3 +29,36 @@ def update_dutch_auction_price_task(auction_id, delta_price):
     auction = DutchAuction.objects.get(pk=auction_id)
     auction.current_price -= delta_price
     auction.save(update_fields=['current_price'])
+
+
+@app.task
+def send_price_change_email_task(lot, email, new_price):
+    subject = 'Price change notification'
+    message = f'The price of the lot "{lot.item.title}" has changed.\n\nOld Price: ${lot.auction.current_price}\nNew ' \
+              f'Price: ${new_price}'
+    from_email = settings.EMAIL_HOST_USER
+    recipient_list = [email]
+
+    send_mail(subject, message, from_email, recipient_list)
+
+
+@app.task
+def send_lot_sold_email_task(lot, email):
+    subject = 'Lot sold notification'
+    message = f'Congratulations! You bought a lot "{lot.item.title}" for ${lot.auction.current_price}.'
```

Comment:
переопредели метод _\_str_\_ у lot и потом сделай
```suggestion
    message = f'Congratulations! You bought a lot {lot!r} for ${lot.auction.current_price}.'
```